src/applications/personalization/profile360/actions/paymentInformation.js

```diff
@@ -1,7 +1,12 @@
 import { getData } from '../util';
+import recordEvent from 'platform/monitoring/record-event';
 
+export const PAYMENT_INFORMATION_FETCH_STARTED =
+  'FETCH_PAYMENT_INFORMATION_STARTED';
 export const PAYMENT_INFORMATION_FETCH_SUCCEEDED =
   'FETCH_PAYMENT_INFORMATION_SUCCESS';
+export const PAYMENT_INFORMATION_FETCH_FAILED =
+  'FETCH_PAYMENT_INFORMATION_SUCCESS';
```

Comment:
I think this is copy and pasted from the action above.

Reply:
Fixed.